lib/index.js

```diff
@@ -1,54 +1,80 @@
-  }
+    function checkFunction(node) {
+      check(node, allowOneLiners, maxInOneLiner);
+    }
 
-  function checkArrow (node) {
-    check(node, true, maxInOneLiner);
-  }
+    function checkArrow(node) {
+      check(node, true, maxInOneLiner);
+    }
 
-  function check (node, allowOneLiners, maxInOneLiner) {
-    var fnBody = node.body;
+    function check(node, allowOneLiners, maxInOneLiner) {
+      var fnBody = node.body;
 
-    // Arrow functions can return literals that span multiple lines
-    if (fnBody.type === 'ObjectExpression' || fnBody.type === 'ArrayExpression') {
-      return;
-    }
+      // Arrow functions can return literals that span multiple lines
+      if (
+        fnBody.type === "ObjectExpression" ||
+        fnBody.type === "ArrayExpression"
+      ) {
+        return;
+      }
 
-    var isBlockBody = fnBody.type === 'BlockStatement';
-    var body = isBlockBody ? fnBody.body : [fnBody];
+      var isBlockBody = fnBody.type === "BlockStatement";
+      var body = isBlockBody ? fnBody.body : [fnBody];
 
-    // Allow empty function bodies to be of any size
-    if (body.length === 0) {
-      return;
-    }
+      // Allow empty function bodies to be of any size
+      if (body.length === 0) {
+        return;
+      }
 
-    var stmt = body[0];
-    var bodyStartLine = stmt.loc.start.line;
-    var openTokenLine = context.getTokenBefore(stmt).loc.start.line;
-    var closeTokenLine = isBlockBody ? context.getTokenAfter(stmt).loc.start.line : context.getLastToken(stmt).loc.start.line;
+      var stmt = body[0];
+      var bodyStartLine = stmt.loc.start.line;
+      var openTokenLine = context.getTokenBefore(stmt).loc.start.line;
+      var closeTokenLine = isBlockBody
+        ? context.getTokenAfter(stmt).loc.start.line
+        : context.getLastToken(stmt).loc.start.line;
 
-    if (allowOneLiners === true &&
+      if (
+        allowOneLiners === true &&
         body.length <= maxInOneLiner &&
-        openTokenLine === closeTokenLine) {
-      return;
-    }
+        openTokenLine === closeTokenLine
+      ) {
+        return;
+      }
 
-    if (bodyStartLine - openTokenLine < 2) {
-      context.report(node, 'Missing blank line at beginning of function.');
+      if (bodyStartLine - openTokenLine < 2) {
+        context.report({
+          node: node,
+          message: "Missing blank line at beginning of function.",
+          fix: function(fixer) {
+            return fixer.insertTextBefore(node.body.body[0], "\n");
+          }
+        });
+      }
     }
-  }
 
-  return {
-    ArrowFunctionExpression: checkArrow,
-    FunctionExpression: checkFunction,
-    FunctionDeclaration: checkFunction
-  };
+    return {
+      ArrowFunctionExpression: checkArrow,
+      FunctionExpression: checkFunction,
+      FunctionDeclaration: checkFunction
+    };
+  }
 };
 
-module.exports.esLintRuleName = 'hapi-scope-start';
+module.exports.esLintRuleName = "hapi-scope-start";
```

Comment:
Please undo any stylistic changes. They are causing the linter to fail (`npm test`).

Reply:
yeah will do, my linter + fixer automatically screws up any project that has no proper .eslint config in their repository :/